packages/vite/src/node/plugins/worker.ts

```diff
@@ -23,7 +24,9 @@ interface WorkerCache {
   emitted: Map<string, string>
 }
 
-const WorkerFileId = 'worker_file'
+export type WorkerType = 'classic' | 'module' | 'ignore'
+
+export const WorkerFileId = 'worker_file'
```

Comment:
I feel that this variable should be:

```suggestion
export const WORKER_FILE_ID = 'worker_file'
```

since it's a constant.